nebula/ui/components/forms/VPNComboBox.qml

```diff
@@ -204,4 +204,15 @@ ComboBox {
             }
         }
     ]
+
+    Connections {
+        target: window
+        function onRemoveFocus(x, y) {
```

Comment:
Comments:
- we could check whether the input has focus and exit out if not before doing the mapping. it seems like this could get called a lot on mobile and I'm wondering how expensive `mapToItem` will be.
- any reason not to consolidate the `onRemoveFocus()` logic into a single fn on main.qml?

Reply:
Good ideas! added